### PR TITLE
You can now defib monkeys

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: You can now defib monkeys and aliums
tweak: Monkeys can also use defibs now, too!
/:cl:

### CODE REVIEW

**Comment by Xhuis** [comment]
I'll defib you

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -455,4 +455,33 @@
...
\ No newline at end of file
+			B.damaged_brain = TRUE
+
+
+/////////////////////////////////////
+//MONKEYS WITH TOO MUCH CHOLOESTROL//
+/////////////////////////////////////
+
+/mob/living/carbon/proc/can_heartattack()
+	CHECK_DNA_AND_SPECIES(src)
```
Comment (XDTM) [inline code/modules/mob/living/carbon/life.dm]: this will return null if the monkey has no species, which is doesn't, so can_heartattack will always be no

Comment (MoreRobustThanYou) [reply code/modules/mob/living/carbon/life.dm]: Thanks

**Comment by duncathan** [comment]
i tried to help but apparently github conflict editor is bad, conflicts pls

**Comment by MoreRobustThanYou** [comment]
I'm on mobile rn. Also try using GitHub fixer twice, usually works

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**